Tests/TuistKitTests/ProjectEditor/ProjectEditorMapperTests.swift

```diff
@@ -179,28 +215,32 @@ final class ProjectEditorMapperTests: TuistUnitTestCase {
         let (project, graph) = try subject.map(tuistPath: tuistPath,
                                                sourceRootPath: sourceRootPath,
                                                xcodeProjPath: xcodeProjPath,
+                                               setupPath: setupPath,
+                                               configPath: configPath,
                                                manifests: manifestPaths,
                                                helpers: helperPaths,
                                                templates: templates,
                                                projectDescriptionPath: projectDescriptionPath)
 
         // Then
         let targetNodes = graph.targets.values.flatMap { targets in targets.compactMap { $0 } }.sorted(by: { $0.target.name < $1.target.name })
-        XCTAssertEqual(targetNodes.count, 2)
+
+        // expecting `targetNodes == [Config, ModuleManifests, Setup, TemporaryDirectory.XXXMainifest]`
+        XCTAssertEqual(targetNodes.count, 4)
         XCTAssertEqual(targetNodes.first?.dependencies, [])
-        XCTAssertEqual(targetNodes.last?.dependencies, [])
+        XCTAssertEqual(targetNodes.last?.dependencies.sorted(by: { $0.name < $1.name }), [targetNodes[0], targetNodes[2]])
```

Comment:
I'd use `Set` here

Reply:
What if `targetNodes.last?.dependencies` contains duplication which I would like to catch in unit tests?